### PR TITLE
add components & basic layout for landing page

### PR DESCRIPTION
**Pull request by snrelghgub**
Our basic page layout has the following React component structure for our App /> as we agreed yesterday:

Navbar />
Header />
Search /> --> SeachBar />
Result /> --> Card />
Footer />
*where SearchBar /> is a child component of Search /> & Card /> is a child component of Result />
**please also note I decided to use the name Card /> instead of Volume /> to separate & isolate our dependency on Google Books API's naming conventions in its data model. 

Thanks in advance for reviewing my PR & providing me with your feedback.

### CODE REVIEW

**Comment by ArunJose** [comment]
Instead of naming the component Results I would suggest to pick a more suitable name like BookCardList/BookCard, BookList/BookListItem, CardList/Card,  BookCards/BookCard etc. We can can probably reuse the same component in other places like when displaying books in favorites. Also, it will more accurately describe what the component is rendering.

**Comment by snrelghgub** [comment]
that's really smart suggestion @ArunJose. I updated to CardList/Card already - I definitely agree that it is a more appropriate name combination to represent its reusability. Thanks for your constructive input 👍

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function Navbar() {
+  return (
+    <footer>
+      <p>*Footer info goes here*</p>
```
Comment (ArunJose) [inline src/components/Footer.tsx:6]: Exported function should be named Footer

**Review by ArunJose — CHANGES_REQUESTED** [review]
Request you to change name of exported component in Footer.tsx

**Review by ArunJose — APPROVED** [review]

**Pull request merged**